ops/terraform/services/pipeline/iam.tf

```
@@ -1,35 +1,42 @@
-resource "aws_iam_user" "this" {
-  count         = local.create_etl_user ? 1 : 0
-  force_destroy = false
-  name          = "bfd-${local.env}-${local.legacy_service}"
-  path          = "/"
-  tags = {
-    Note    = "NoRotate"
-    Purpose = "ETL PUT"
-    UsedBy  = "CCW"
-  }
-}
-
-resource "aws_iam_access_key" "this" {
-  count = local.create_etl_user ? 1 : 0
-  user  = aws_iam_user.this[0].name
+locals {
+  ccw_rif_role_principal_arns = [
+    for arn in split(" ", local.sensitive_ccw_service_config["ccw_rif_role_principal_arns"]) : trimspace(arn)
+  ]
+  ccw_rif_role_external_id = local.sensitive_ccw_service_config["ccw_rif_role_external_id"]
 }
 
-resource "aws_iam_group" "this" {
-  count = local.create_etl_user ? 1 : 0
-  name  = "bfd-${local.env}-${local.legacy_service}"
-  path  = "/"
+resource "aws_iam_role" "ccw_rif" {
+  count = local.is_prod ? 1 : 0
+  name = "bfd-${local.env}-ccw-rif"
+  description = "Role assumed by CCW to read and write to the ${local.env} ETL bucket."
```

I think we can continue avoiding comments here if we expand the description just a bit, maybe something like:
```suggestion
  description = "Role assumed by CCW to read and write to the ${local.env} production and verification ETL buckets."
```